### PR TITLE
[FEAT] updated the title page with a more accurate description

### PR DESCRIPTION
**Pull request by jakeleveroni**

### CODE REVIEW

File: react-starter-app/src/App.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ class App extends Component {
       <div className="App">
         <div className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+          <h2>Welcome Team 05</h2>
         </div>
         <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
+          We need a better name...<code>src/App.js</code> and save to reload.
```
Comment (chantellesuarez) [inline react-starter-app/src/App.js]: I agree

File: react-starter-app/src/App.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ class App extends Component {
       <div className="App">
         <div className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+          <h2>Welcome Team 05</h2>
```
Comment (lilipach) [inline react-starter-app/src/App.js]: who came up with this name?

Comment (lykamarcelino) [inline react-starter-app/src/App.js]: This is my comment

File: react-starter-app/src/App.js (hunk 1 above)
Comment (chantellesuarez) [inline react-starter-app/src/App.js]: I agree

**Pull request merged**